lib/src/react16_suggestors/react_style_maps_updater.dart

```diff
@@ -39,36 +40,40 @@ class ReactStyleMapsUpdater extends GeneralizingAstVisitor
 
           /// CSS properties that were modified by the script or need to be
           /// manually checked
-          List<String> affectedValues = [];
+          List<String> listOfVariables = [];
 
           // Variables that affect the writing of the comment that goes above
           // the style prop. There are different messages patched based on
           // these booleans.
           bool styleMapContainsAVariable = false;
           bool isAVariable = false;
           bool isAFunction = false;
+          bool isUnknown = false;
           bool isForCustomProps =
               cascade.toString().toLowerCase().contains('props');
 
           if (stylesObject is MapLiteral) {
             stylesObject.entries.forEach((MapLiteralEntry cssPropertyRow) {
-              String originalCssPropertyKey =
-                  cssPropertyRow.beginToken.toString();
-              String cleanedCssPropertyKey =
-                  cleanString(originalCssPropertyKey);
-
-              // We cannot simply do `cssPropertyRow.endToken` because if the
-              // "end token" is an expression, it only returns the last
-              // value in the ternary. Removing the beginning token keeps the
-              // ternary intact.
-              List endToken = cssPropertyRow.childEntities.toList();
-              endToken.removeRange(0, 2);
-              String originalCssPropertyValue = endToken.join(" ");
-              String cleanedCssPropertyValue =
-                  cleanString(originalCssPropertyValue);
-
-              bool rowContainsAVariable =
-                  nodeIsLikelyAVariable(originalCssPropertyValue);
+              final propertyKey = cssPropertyRow.key;
+              String originalCssPropertyKey = propertyKey.toString();
```

Comment:
#nit Should this be `toSource`? (I'm asking because I'm not sure if `toString` is documented as having the desired behavior)

Reply:
`toSource` definitely sounds like the preferred method. Thank you for pointing that out! I'm learning a lot about the AST API's